src/test/java/ru/training/at/hw3/pages/DifferentElementsPage.java

```diff
@@ -0,0 +1,83 @@
+package ru.training.at.hw3.pages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+import java.util.List;
+
+public class DifferentElementsPage {
+    private WebDriver driver;
```

Comment:
If we create a common abstract class for pages we can use common variables and methods